tensorflow/compiler/tf2tensorrt/BUILD

```diff
@@ -384,6 +384,9 @@ tf_gen_op_libs(
         "get_calibration_data_op",
         "trt_engine_resource_ops",
     ],
+    hdrs = [
+        "common/utils.h",
```

Comment:
this cause build failure when building ops/trt_engine_op.cc for example, because common/utils.h requires nvinfer.h which is not in the dependence. It would be nice if you can avoid this change and thus the change in tensorflow.bzl.

Reply:
Actually, I don't need this change if and when [PR#57568](https://github.com/tensorflow/tensorflow/pull/57568#issuecomment-1233415865) will be merged. Should I wait?

Reply:
@bixia1: I rolled back changes related to build scripts, because after the merge  [PR#57568](https://github.com/tensorflow/tensorflow/pull/57568) I don't need them. PR is ready for merge.